engines/sci/detection.cpp

```diff
@@ -784,6 +785,8 @@ SaveStateList SciMetaEngine::listSaves(const char *target) const {
 	return saveList;
 }
 
+bool SciMetaEngine::simpleSaveNames() const { return true; }
```

Comment:
SCI _usually_ uses standard savegame names, but not for all games, and some games additionally store other files using the savegame system (containing settings and such).